clients/drcachesim/tracer/instru.h

```diff
@@ -292,6 +292,10 @@ class instru_t {
     get_cpu_id();
     static uint64
     get_timestamp();
+    static uint64
+    get_attached_timestamp();
```

Comment:
Put up by the frozen timestamp routine, and make the variable a member variable like the frozen timestamp?  See the same suggestion in another comment.  It's not really related to get_timestamp().